### ISSUE TITLE
type="date" and class="form-control" in Android "Browser" <= 4.3

### ISSUE THREAD

**Issue by nmaya** [twbs/bootstrap#16898]
In Android "Browser" <= 4.3,
when a input is type="date" and class is "form-control",
caret (or I-beam) position and text position are shifted down.

http://jsfiddle.net/xnvf5k37/

When a input is type="text", this issue doesn't occur.

In Android "Browser" >= 4.4, this issue doesn't occur.

![cap1](https://cloud.githubusercontent.com/assets/5668936/8923849/46dc6d72-3531-11e5-9b7a-ab81c25a57a5.png)

![cap2](https://cloud.githubusercontent.com/assets/5668936/8923853/509be2c0-3531-11e5-87d7-2bc2f4ee679a.png)

**Comment by twbs-lmvtfy** [comment]
Hi @nmaya!

You appear to have posted a live example (http://fiddle.jshell.net/xnvf5k37/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 42, column 9 thru line 42, column 116: Attribute ```` placeholder ```` is only allowed when the input type is ```` email ````, ```` number ````, ```` password ````, ```` search ````, ```` tel ````, ```` text ````, or ```` url ````.
* line 48, column 9 thru line 48, column 95: Attribute ```` placeholder ```` is only allowed when the input type is ```` email ````, ```` number ````, ```` password ````, ```` search ````, ```` tel ````, ```` text ````, or ```` url ````.
* line 51, column 5 thru line 51, column 54: Duplicate ID ```` form-group_hoge_date ````.
* line 57, column 5 thru line 57, column 54: Duplicate ID ```` form-group_hoge_text ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Please note that we do not officially support the non-Chrome-based Android Browser, due to shit like this.

**Comment by mdo** [comment]
Probably okay to close as a won't fix.
